contracts/market/RepayFacet.sol

```diff
@@ -353,7 +362,8 @@ library RepayLib {
 
         // Calculate available collateral for reward
         if (collateralAmount > 0) {
-            collateralValue_ = AppStorageLib.store()
+            collateralValue_ = AppStorageLib
+                .store()
```

Comment:
this linting change should not have happened

please make sure you have ran `yarn install` and then add another commit after you run `yarn format:write`